easybuild/easyconfigs/e/EGSnrc/log-dir_finanlize.patch

```diff
@@ -0,0 +1,11 @@
+--- HEN_HOUSE/scripts/finalize_egs_foruser.orig	2018-10-30 15:50:18.278569800 +0100
```

Comment:
Please add description of the patch (even if it is this trivial), and author (with date)

Reply:
Done, thank you for the feedback